apps/allocations/test/allocations.test.js

```diff
@@ -4,7 +4,8 @@ const {
   DAOFactory,
   EVMScriptRegistryFactory,
   Kernel,
-  MiniMeToken
+  MiniMeToken,
+  StandardBounties
```

Comment:
Let's remove this unneeded import, allocations is not going to use StandardBountes, please, look at the correct `Allocations.test.js` file in `dev` branch and rebase from it before messing up with further changes